package.json

```diff
@@ -3,6 +3,13 @@
   "version": "0.0.1",
   "license": "Apache-2.0",
   "description": "Shared ESLint configurations from the Oisy Wallet team",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dfinity/eslint-config-oisy-wallet.git"
+  },
+  "bugs": {
+    "url": "https://github.com/dfinity/eslint-config-oisy-wallet"
```

Comment:
shall this point directly to `/issues`

Reply:
I actually never do so, don't know why. That's a good point. We can.

Reply:
Done.